docker/sf-gpu.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM tensorflow/tensorflow:2.10.1-gpu
+RUN pip install -U secretflow -i https://pypi.doubanio.com/simple/
+RUN pip install jupyter -i https://pypi.doubanio.com/simple/
+
+VOLUME  /tf/
+EXPOSE 8888
```

Comment:
Remove expose by the way. We do not expose a fix port for user if not very necessary, it may become vulnerable. 
You can guide user how to expose port when running docker container, and  make it clear that user can choose a random unused port.